resources/quality/goofoo/petg/goofoo_open_0.40_petg_standard.inst.cfg

```
@@ -0,0 +1,18 @@
+[general]
+version = 4
+name = Standard Quality
+definition = goofoo_open
+
+[metadata]
+setting_version = 16
+type = quality
+quality_type = normal
+material = generic_petg
```

Did you mean goofoo_petg?
```suggestion
material = goofoo_petg
```

---

I have already edited it